src/coreclr/tools/ILVerification/InstantiatedGenericParameter.cs

```diff
@@ -76,6 +76,8 @@ private InstantiatedGenericParameter(GenericParameterDesc genericParam)
 
         public override GenericConstraints Constraints => _genericParam.Constraints;
 
+        public override TypeSystemEntity AssociatedTypeOrMethod => _genericParam.AssociatedTypeOrMethod;
```

Comment:
I don't know much about the ILVerify codebase, but doesn't this need `InstantiateSignature`?

Reply:
Maybe? I don't know that codebase either, but it doesn't seem unreasonable.